recipe-server/client/control_new/utils/forms.js

```diff
@@ -139,6 +139,8 @@ export function createForm({ validateFields, ...formConfig }) {
  */
 export function connectFormProps(Component) {
   return class Wrapper extends React.Component {
+    static wrappedComponent = Component;
```

Comment:
I don't think we should do this. `connect` already uses a similar variable name, plus using the `static` method should remove the need for this entirely.

Reply:
I don't agree. Although it isn't needed in this case, I think that any sort of wrapper component should provide access to the wrapped component, for ease of testing. I suggested @marcrowo use `wrappedComponent` specifically because it matched what `connect` uses.